src/cache.js

```diff
@@ -59,5 +61,6 @@ export function removeOne(orgName) {
   }
   catch {
     // Do nothing, file doesn't exists.
+    logger.error("Failed to update orgs.json");
```

Comment:
same, catch error and log

Reply:
I think it could even be:
```suggestion
    logger.error(`Organisation ${orgName} does not exists.`);
```
We don't need to log the error, the file just doesn't exists